R/input-select.R

```diff
@@ -212,3 +212,121 @@ selectizeIt <- function(inputId, select, options, nonempty = FALSE) {
 
   attachDependencies(select, selectizeDep)
 }
+
+
+
+
+
+
+
+
+#' Create a select list input control from a data.frame
+#'
+#' Create a select list that can be used to choose a single or multiple items
+#' from the column names of a data frame.
+#'
+#' The resulting output value will be returned as a symbol or a list of symbols
```

Comment:
... returned as a symbol (if multiple = FALSE) or a list of symbols (if multiple = TRUE)...

Or maybe make a bulleted list? Then you could explain `!!` vs `!!!` for each bullet too.